Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <sstream>
+#include <string>
+
+
+#include "Person.h"
+
+
+Person::Person(){
```

Comment:
Our code formatting will have the curly brace on a line. Example:
```
Person::Person()
{
  age_ = new int(0);
}
```

Reply:
Remember to change the rest of the file.

Reply:
@DatLam25 We should include this in our [style guide.](https://github.com/UCSolarCarTeam/Development-Resources/tree/master/Development-Guide/Style-Guide)